.github/workflows/deploy-gcp-tests.yml

```diff
@@ -250,6 +263,18 @@ jobs:
           --mount type=volume,src=${{ inputs.disk_prefix }}-${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }},dst=${{ inputs.root_state_path }}/${{ inputs.zebra_state_dir }} \
           ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}"
 
+      # Create image from disk that will be used for following tests
+      # Force the image creation as the disk is still attached even though is not being used by the container
+      - name: Create image from state disk
```

Comment:
Question: Does this disk just contain lightwalletd state, or both zebra and lightwalletd state?

Can you document that in the comments?

Reply:
Both

Reply:
I'm not sure if we want to put both states in the same directory on the disk, or if we even can, so I'm re-opening this conversation.

Reply:
The states are being saved in different directories:
- `/var/cache/zebrad-cache/../..` and
- `/var/cache/lwd-cache/../..`